addon/components/item-picker/feature-service-preview/component.js

```diff
@@ -22,7 +22,7 @@ export default Component.extend({
   classNames: [ 'item-picker-current-item-preview' ],
   description: reads('model.item.description'),
   featureService: service('feature-service'),
-  forceLayerSelection: alias('showLayers'),
+  forceLayerSelection: alias('params.forceLayerSelection'),
```

Comment:
I think it is better to use `reads` unless you intend to call `this.set('forceLayerSelection)` but that would break DDAU.